.gitignore

```diff
@@ -2,6 +2,7 @@
 
 ### Python artifacts
 *.pyc
+migrations/
```

Comment:
Remove this folder from .gitignore. Migrations should always be committed alongside the code.

Reply:
done in the latest commit.